book/src/dev/rfcs/0008-release-planning.md

```diff
@@ -0,0 +1,190 @@
+- Feature Name: `release_planning`
+- Start Date: 2020-09-14
+- Design PR: [ZcashFoundation/zebra#0000](https://github.com/ZcashFoundation/zebra/pull/0000)
+- Zebra Issue: [ZcashFoundation/zebra#0000](https://github.com/ZcashFoundation/zebra/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Release and distribution plans for Zebra.
+
+# Motivation
+[motivation]: #motivation
+
+We need to plan our release and distribution processes for Zebra.  Since these
+processes determine how users get Zebra and place constraints on how we do
+Zebra development, it's important to think through the implications of our
+process choices.
```

Comment:
Notes from a discussion on updates and maintenance:

Streamlining updates
- encourage everyone to use latest
- stability commitment so updates are lower risk
- breaking changes are easier if they are small and frequent
- maybe we sometimes want to create legacy compatibility modes - but they come with a maintenance cost

Cost of maintaining even one stable release
- it's harder to do refactors, because backports need to be compatible with both versions
- maybe we want to maintain the old and new versions around a network upgrade?